python/python/tests/test_transaction.py

```diff
@@ -763,3 +763,24 @@ async def test_transaction_lastsave(
         lastsave_time = response[0]
         assert isinstance(lastsave_time, int)
         assert lastsave_time > yesterday_unix_time
+
+    @pytest.mark.parametrize("cluster_mode", [True])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_lolwut_transaction(self, redis_client: RedisClusterClient):
+        transaction = Transaction()
+        transaction.lolwut().lolwut(5).lolwut(parameters=[1, 2]).lolwut(6, [42])
+        results = await redis_client.exec(transaction)
+        assert results is not None
+
+        if isinstance(results, dict):
+            for node_result in results.values():
+                if isinstance(node_result, str) and "Redis ver. " in node_result:
+                    assert True
+                else:
+                    assert False
+        else:
+            for element in results:
+                if isinstance(element, str) and "Redis ver. " in element:
+                    assert True
+                else:
+                    assert False
```

Comment:
Similarly to the other tests, we know that we should not get a dict here because transaction should be sent to a single node, so we don't need the if-else
```suggestion
        for element in results:
            assert "Redis ver. " in element
```

Reply:
I may need to keep this `if isinstance(element, str)` otherwise CI/CD will fail

Reply:
If it is not a `str`, so it should be a cluster response `Dict[str, str]` or so. So you should check all values in that `Dict`.

Reply:
This is a transaction, so it shouldn't be a cluster response I don't think